ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/AdminLogsRepairCache.java

```diff
@@ -51,7 +51,9 @@ public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
         blockchainQueries
             .getTransactionLogBloomCacher()
             .orElseThrow(
-                () -> new InternalError("Error attempting to get TransactionLogBloomCacher"));
+                () ->
+                    new InternalError(
+                        "Error attempting to get TransactionLogBloomCacher. Please ensure that the TransactionLogBloomCacher is enabled."));
```

Comment:
This is good! It covers the case where we have an error while also addressing the most likely case